src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+@import "./utils/variables";
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 50px 0;
+  margin: auto;
+  max-width: 942px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 0 16px 16px;
+  margin-bottom: 48px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

You created a variables for colors, but not for all 🥺 
```suggestion
  border: 1px solid $c-color;
```